src/currencyexchange/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:10.16.1-alpine
```

Comment:
this is fine for now. (I used something similar for destination as well)

We will need to optimize the image more later.